### PR TITLE
C-4 -  Endpoint for github project cloning 

### PR DESCRIPTION
**Pull request by ban17**
# What is done
* added .env
* mysql connect
* adding repositories to the database
* script for cloning repos
* script for getting list of commits and files
* saving files in folders with name of commit (just changed files are saved, so it's ready for importing to the database)
* table commits table changed to have id_repository instead of id_tutorial

Probably this need to be continued in C-12 task

### CODE REVIEW

**Comment by janko1031** [comment]
Just small thing, before starting with review.
Readme docs says:
npm run instread of 'start'

File: scripts/add-files-and-changes.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+repo='/guide/codeguides-server/repos/4'
```
Comment (janko1031) [inline scripts/add-files-and-changes.sh]: can you make this part dynamic?
I for example, have project in different dir.

File: scripts/add-files-and-changes.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+cd $repo || exit
+commits=$(git log --reverse --pretty=format:"%h")
+
+id=${repo##*/}
+parsedDir=/guide/codeguides-server/parsed/$id
```
Comment (janko1031) [inline scripts/add-files-and-changes.sh]: same here?

**Review by ZarkoZivojnovic — APPROVED** [review]
Looks good to me :+1:  (at least the part I understand) :D

**Review by janko1031 — APPROVED** [review]
looks good to me

**Review by freeman-tmc — APPROVED** [review]

**Pull request merged**